both/routes/user.js

```diff
@@ -30,6 +30,8 @@ Router.map(function () {
     template: 'userEdit',
     waitOn: function () {
       Meteor.subscribe('users');
+      Meteor.subscribe('userClasses');
```

Comment:
Forgot to update this. This was from when I had the two collections, userClasses and classes. Nice catch.

Resolved.